src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,46 @@
+package duke.command;
+
+import duke.Duke;
+import duke.DukeException;
+import duke.Parser;
+
+/**
+ * This class encapsulates a user's command.
+ */
+public abstract class Command {
+    public static final Command[] COMMANDS = new Command[] { new DeadlineCommand(), new DeleteCommand(),
+        new DoneCommand(), new EndCommand(), new EventCommand(), new FindCommand(), new ListCommand(),
+        new TodoCommand() };
```

Comment:
Indentation for wrapped lines should be 8 spaces.

Reply:
Thanks for spotting this, fixed in a later commit!